### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by BezzB**
Project requirements
At this point you should have a fully working app that uses only basic styles to make the layout work, according to this wireframe:
Basic UI

Now you should improve the look and feel of the application, adding the styles of your choice.
Please keep the general layout of the wireframe, this is the only mandatory requirement.
You can use plain CSS or any CSS framework, it's up to you.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @BezzB, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove all section descriptions like in the image below remove this text: "Mention all of the collaborators of this project.". You have similar descriptions in the Show you support and Acknowledgement sections.

image

### Kindly remove this dummy part of the readme file also.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -14,25 +14,22 @@
...
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [LeaderBoard-Setup-ES6](#leaderboard-setup-es6)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[LeaderBoard-Setup \] ](#-leaderboard-setup--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
```
Comment (rotimiazeez) [inline README.md:23]: Please note that in our live demo section, the provided link does not match the current project. 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @BezzB 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good app UI 👏🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖

You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (prolajumokeoni) [inline README.md:23]: - [ ] It would be better to remove the live demo  link that is not related to this project, for this milestone you don't need to deploy the application 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @BezzB 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
 - Application styled like wireframe 👏🏿  

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @BezzB 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Descriptive PR ✔️
- Nice use of code semantics✔️
- Great UI ✔️

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
